### PR TITLE
Updated message from config check of unconnected inputs

### PR DESCRIPTION
**Pull request by naylor-b**
- now includes units and values
- any cases where the values for connected inputs don't match are highlighted

### CODE REVIEW

File: openmdao/error_checking/check_config.py [hunk 1]
```diff
@@ -218,20 +291,55 @@ def _check_hanging_inputs(problem, logger):
...
+                                                 absnames, units, vals)
+                    if mismatch:
+                        msg.append("\n   ----- WARNING: inconsistent units and/or values!! -----\n")
+                    msg.append("   {}  (p):\n".format(prom))
+                    for a, u, v in zip(absnames, units, vals):
+                        msg.append("      " +
+                                   template_prom.format(a, u,
+                                                        _trim_str(problem.get_val(a,
+                                                                                   get_remote=True),
+                                                                   25),
```
Comment (Kenneth-T-Moore) [inline openmdao/error_checking/check_config.py]: This is really hard to read. Could you create a temporary variable to hold `_trim_str(problem.get_val(a,get_remote=True)` to improve readability? Added advantage, looks like you can do it outside of the if branch.

File: openmdao/error_checking/check_config.py [hunk 2]
```diff
@@ -218,20 +291,55 @@ def _check_hanging_inputs(problem, logger):
...
-                msg.append("   %s\n" % prom)
-            else:  # promoted
-                msg.append("   %s: %s\n" % (prom, prom_ins[prom]))
+        with printoptions(precision=3):
+            for prom, absnames, units in sorted(unconns, key=lambda x: x[0]):
+                if len(absnames) == 1 and prom == absnames[0]:  # not really promoted
+                    a = absnames[0]
+                    msg.append("   " +
+                               template_abs.format(a, units[0],
+                                                   _trim_str(problem.get_val(a, get_remote=True),
```
[the remark this answers is not in the document]
Comment (Kenneth-T-Moore) [reply openmdao/error_checking/check_config.py]: i don't know, maybe we should do both truncated and norm...

[the remark this answers is not in the document]
Comment (naylor-b) [reply openmdao/error_checking/check_config.py]: ok I updated it to use the norm if the string exceeds the specified width.

File: openmdao/error_checking/check_config.py (hunk 1 above)
Comment (naylor-b) [reply openmdao/error_checking/check_config.py]: fixed

**Pull request merged**